spec/system/department_admin_manages_newsletters_spec.rb

```diff
@@ -0,0 +1,238 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Admin manages newsletters', type: :system do
+  let(:organization) { create(:organization) }
+  let(:area) { create(:area, organization: organization) }
+  let(:department_admin) { create(:department_admin, :confirmed, name: 'Sarah Kerrigan', organization: organization, area: area) }
+  let!(:deliverable_users) { create_list(:user, 5, :confirmed, newsletter_notifications_at: Time.current, organization: organization) }
+
+  before do
+    switch_to_host(organization.host)
+    login_as department_admin, scope: :user
+  end
+
+  describe 'creates and previews a newsletter' do
```

Comment:
prefer `context` instead of `describe` inside `describe` blocks.

Reply:
Ok I see that you use `describe` to group inside the file. But then you have to be consistent and use always `describe` for first level blocks.

Reply:
Done!

Reply:
I think we did not understand each other, but well...

Reply:
May be... What do you want to do then?